webapp/components/create_comment.jsx

```diff
@@ -576,7 +574,7 @@ export default class CreateComment extends React.Component {
                                 value={this.state.message}
                                 onBlur={this.handleBlur}
                                 createMessage={Utils.localizeMessage('create_comment.addComment', 'Add a comment...')}
-                                emojiEnabled={this.state.emojiPickerEnabled}
+                                emojiEnabled={window.mm_config.EnableEmojiPicker === 'true'}
```

Comment:
That's close to what I wanted, but I actually meant that we could remove the `emojiEnabled` props from the `Textbox` and `FileUpload` components, and just look at mm_config in both of them